scripts/extract-changelog.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
I prefer this to be written in one of the scripting languages so you can add some unit tests for it.